CHANGES.md

```diff
@@ -8,6 +8,10 @@
   https://github.com/cnpem/epics-in-docker/pull/42
 * ci: add reusable job for IOC images. by @ericonr in
   https://github.com/cnpem/epics-in-docker/pull/49
+* ioc: copy locally installed software from build stages. by @henriquesimoes in
+  https://github.com/cnpem/epics-in-docker/pull/55
+  * This allows the IOC build phase to install custom files, available at
+    runtime, under `/usr/local` tree for all targets.
```

Comment:
I don't think "locally installed" is clear enough, because distro packages are still locally installed, and for those we should be using the package installation mechanism.

The same goes for the commit message.

Reply:
> I don't think "locally installed" is clear enough

I understand. I followed the terminology used by FHS itself:

> Locally installed software must be placed within /usr/local rather than /usr unless it is being installed to replace or upgrade software in /usr. [(In 4.9 Local hierarchy)](https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch04s09.html#ftn.idm236091898144)

Any suggestion?

Reply:
so that's done already, great.